### ISSUE TITLE
Add documentation for use with fzf-lua nvim plugin

### ISSUE THREAD

**Issue by ibhagwan** [skim-rs/skim#418]
Not really an issue, I saw that you are maintaining [`skim.vim`](https://github.com/lotabout/skim.vim), based off of `fzf.vim`.

I recently wrote [`fzf-lua`](https://github.com/ibhagwan/fzf-lua), as you can probably guess from the name it's an implementation of `fzf.vim` in pure lua, with many more features (icons, git indicators, native LSP support and more).

Yesterday I saw your project and due to my soft spot for any rust implementation of projects I love I decided to add support for `skim` (which is amazing btw), due to the great compatibility of the command line options it only took minor modifications to make it work and one slightly bigger modification to be able to use "interactive mode" for live grep (rg), which was utilizing the `change:reload` event in `fzf`.

If you'd like to try it out, follow the installation instructions on [my repo](https://github.com/ibhagwan/fzf-lua), ~~note that until [@vijaymarupudi](https://github.com/vijaymarupudi) decides what to do [with this PR](https://github.com/vijaymarupudi/nvim-fzf/pull/16) you need to use [my fork of nvim-fzf](https://github.com/ibhagwan/nvim-fzf)~~ and just change the `fzf_bin` variable to `sk` instead.

In short, add the below plugins (using `packer.nvim`):
```lua
use { 'ibhagwan/fzf-lua',
  requires = { 'nvim-tree/nvim-web-devicons' } -- optional for icons
}
```
or using `vim-plug`:
```vim
Plug 'ibhagwan/fzf-lua'
Plug 'nvim-tree/nvim-web-devicons'
```

And setup the plugin with skim:
```lua
lua << EOF
require'fzf-lua'.setup({fzf_bin = 'sk'})
EOF
```

Here's a glimpse of `fzf-lua` working with `skim` (left side live grep, right side LSP diagnostics):

![screenshot-1627013466](https://user-images.githubusercontent.com/59988195/126738938-5e065bc1-ed77-410f-9ec0-4148b2ad7340.png)

**Comment by stevenroose** [comment]
I just tried to use fzf-lua with skim and it doesn't work. Many of the winopts flags, even though `skim` documents having them, don't work with fzf-lua.

It would be amazing if someone well-versed in skim and neovim's lua extensions would look into how skim and neovim could work better together. Either by improving skim/fzf-lua interactions or having a skim-lua.

**Comment by LoricAndre** [comment]
Thank you very much for your work, I recently configured fzf-lua to work with the current version of skim. I noticed a few issues, but I hope once we solve them on our side we can work together to better integrate skim with fzf-lua (especially the `tmux` variant, as the default usage should work perfectly fine as soon as we readd `index` to the tiebreak options)

**Comment by ibhagwan** [comment]
> I’m excited to try the new `reload` action with fzf-lua

@LoricAndre, unfortunately implmenting this properly requires also the `unbind`, `rebind` and `load` triggers (to be able to unbind the keybind during execution, for example when doing a git stage/unstage, if we don't unbind-execute-rebind, the user might press the bind multiple times and err out.

On another note, if full feature partity with fzf is planned, I can list out all fzf features currently used by fzf-lua which are missing in skim (e.g. info styles, triggers, etc) - let me know if this would help?

**Comment by LoricAndre** [comment]
I'm not sure ATM if we want to aim for full parity or not, as FZF is awesome but I'd still like to keep some freedom.

If you have a list of such features off the top of your head, it would still be extremely useful, as they are currently scattered among the 100+ issues currently opened in the repo

**Comment by LoricAndre** [comment]
@ibhagwan shouldn't we use the `skim` profile instead of manually set `sk_bin` in our doc ? I'm working on updating the README to add your (amazing) plugin

**Comment by ibhagwan** [comment]
> Thank you very much for taking the time to write this, it should be very useful

Forgot to mention a couple of things, will edit my comment to add missing highlights and scroll bar.
